mongo_connector/doc_managers/mongo_doc_manager.py

```diff
@@ -114,9 +113,6 @@ def _get_meta_collection(self, namespace):
     def _namespaces(self):
         """Provides the list of namespaces being replicated to MongoDB
         """
-        if self.namespace_set:
-            return self.namespace_set
-
         user_namespaces = []
```

Comment:
Good catch! After digesting the code a bit more I think it makes more sense to return the meta collection names instead of the replicated collection names.